src/legacy/core_plugins/data/public/query/query_bar/components/query_bar_input.tsx

```diff
@@ -468,11 +468,22 @@ export class QueryBarInputUI extends Component<Props, State> {
       <EuiOutsideClickDetector onOutsideClick={this.onOutsideClick}>
         <div
           style={{ position: 'relative' }}
-          role="combobox"
+          {...{ role: 'combobox' }} // using awkward syntax to avoid triggering false jsx-a11y/role-has-required-aria-props linting failure
+          aria-label={this.props.intl.formatMessage(
```

Comment:
Since we're anyway touching this line and trying to move away from `props.intl` let's just use `i18n.translate` here :-)